src/lib/data/banners/events/3.5.json

```diff
@@ -69,6 +69,71 @@
 					}
 				}
 			}
+		},
+		{
+			"phase": 2,
+			"banners": {
+				"standardVersion": 3,
+				"events": {
+					"item": [
+						{
+							"name": "the-transcendent-one-returns-2",
+							"character": "shenhe",
+							"buttonBoxPosition": {
+								"w": 160,
+								"t": 22
+							},
+							"textOffset": {
+								"t": 78.5,
+								"l": 43
+							}
+						},
+						{
+							"name": "the-heron_s-court-3",
+						"character": "kamisato-ayaka",
+							"buttonBoxPosition": {
+								"w": 95,
+								"t": 32,
+								"l": 50
+							},
+							"textOffset": {
+								"l": 51
+							}
+						}
+					],
+
+					"rateup": ["diona", "sucrose", "mika"]
+				},
+				"weapons": {
+					"name": "epitome-invocation-44",
+					"fatepointsystem": true,
+					"featured": [
+						{
+							"name": "calamity-queller",
+							"type": "polearm",
+							"buttonBoxPosition": {
+								"t": 15,
+								"w": 90,
+								"l": 60
+							}
+						},
+						{
+							"name": "mistsplitter-reforged",
+							"type": "sword",
+							"buttonBoxPosition": {
+								"t": 5,
+								"w": 80,
+								"l": 35
+							}
+						}
+					],
+					"rateup": ["the-bell", "favonius-warbow", "sacrificial-sword", "wine-and-song", "favonius-lance"],
```

Comment:
The first order of the rateup weapons is Wine and Song

![image](https://user-images.githubusercontent.com/13815468/225625657-3f44ace9-563f-4452-b671-839603de2eef.png)